src/app/components/play-round/play-round.component.ts

```diff
@@ -158,6 +156,8 @@ export class PlayRoundComponent implements OnInit {
   }
 
   async clickInteract() {
+    this.interactionInProgress = true;
```

Comment:
It might be technically possible for a use to click the interact button, and then hit it again before this assignment finishes,

but I think that for practical purposes, setting this flag should work all right ☺️

Thanks for setting that up, Paul!

Reply:
(Please excuse me, I'm just always a bit paranoid about async code 😛)

Reply:
I don't think there's really any faster way to disable the button upon clicking it. I suppose we could add some sort of debounce in addition to this but in my testing it seems extremely unlikely to be an issue.